scripts/cnn_variant_wdl/happy_plot.R

```diff
@@ -0,0 +1,79 @@
+library(dplyr)
```

Comment:
I'm not going to review your R code, but if you want another set of eyes I'm sure one of the other Palantir folks would help out.

Reply:
we took a look during a palantir meeting.  the current state is the result of those suggestions.